.vale/styles/foreman-documentation/Capitalization.yml

```diff
@@ -0,0 +1,30 @@
+# Suggest proper capitalization
+---
+extends: substitution
+ignorecase: false
+level: error
+message: "Use '%s' instead of '%s' unless you are using the term in a context that requires different capitalization."
```

Comment:
context means if you describe the Foreman WebUI such as buttons, settings, or templates.

Your call if you want to add this.